infrastructure/scripts/generate-changelog.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set eu -pipefail
+
+time docker run -it --rm -v "$(pwd)":/usr/local/src/your-app githubchangeloggenerator/github-changelog-generator \
```

Comment:
Do we need `time`? 

Also please use a fixed image version instead of latest.

Reply:
done